GitCommands/Remotes/ConfigFileRemoteSettingsManager.cs

```diff
@@ -403,22 +403,38 @@ public void ToggleRemoteState(string remoteName, bool disabled)
             module.LocalConfigFile.Save();
         }
 
-        // pass the list in to minimise allocations
-        private void PopulateRemotes(List<ConfigFileRemote> allRemotes, bool enabled)
+        private List<ConfigFileRemote> PopulateEnabledRemotes()
         {
             var module = GetModule();
+            List<ConfigFileRemote> allRemotes = new List<ConfigFileRemote>();
+            const bool enabled = true;
 
-            Func<IReadOnlyList<string>> func;
-            if (enabled)
-            {
-                func = () => module.GetRemoteNames();
-            }
-            else
+            var remotes = ThreadHelper.JoinableTaskFactory.Run(async () => await module.GetRemotesAsync()).ToList();
```

Comment:
Do we need to `ToList` here?

Reply:
no, removing
(still looking at the tests)